test/ocamlorg_web/graphql_test.ml

```diff
@@ -145,47 +230,113 @@ let state_test () =
 let () =
   Alcotest.run
     "ocamlorg"
-    [ ( "test that starts_with function compares two similar strings"
+    [ ( "test that get_info function returns all the dependencies"
```

Comment:
Also as a minor point, I appreciate the verbosity in these strings but they're perhaps a little too verbose 😆 . 

If you look at the Alcotest examples https://github.com/mirage/alcotest they are (a) smaller and (b) don't contain white spaces. This is pretty useful because the strings are used in the filenames of the output per test case so when something fails and you need to go digging in the output it makes it a little easier to get to in my experience.

Reply:
FWIW, I tend to write pretty verbose test descriptions as well, probably a habit I got from Elixir (one test suite I have in mind is https://github.com/aaronrenner/phx_gen_auth/blob/master/priv/templates/phx.gen.auth/auth_test.exs). But in this instance, I'd agree with Patrick that this could be a bit shorter (e.g. "get_info returns all the dependencies")